migrations/20191206083531_initBorgerdata.js

```diff
@@ -0,0 +1,38 @@
+
+exports.up = async function(knex) {
```

Comment:
Is this table used only for test/development or are we caching data from the real SQL Server?

Reply:
Yup, they are only for test/development. I've updated the README with a little bit of info about this